src/playground/index.ejs

```diff
@@ -2,6 +2,17 @@
 <html>
   <head>
     <meta charset="UTF-8">
+
+    <meta name="twitter:card" content="summary"/>
+    <meta name="twitter:site" content="Programmieren mit der Maus"/>
+    <meta name="twitter:creator" content="wdrmaus.de"/>
```

Comment:
`site` and `creator` should contain usernames for twitter. You did this further down, but fields should not be duplicated